test/check-storage-cockpit

```diff
@@ -68,6 +74,18 @@ class TestStorageCockpitIntegration(VirtInstallMachineCase, StorageCase):
 
     @nondestructive
     def testEncryptedUnlock(self):
+        """
+        Description:
+            Test using for the installationi an encrypted partition created in Cockpit storage
```

Comment:
A typo "installationi", the sentence sounds broken.

Reply:
It's italian installation :P